jobs/validate-kubeflow-microk8s/Jenkinsfile

```diff
@@ -55,24 +55,26 @@ pipeline {
                 exec "sudo snap install lxd"
                 exec "sudo snap install microk8s --classic --channel ${params.microk8s_channel}"
```

Comment:
We should be testing latest/edge right?

Reply:
With LXC in the mix we'll be able to easily test whatever version of microk8s we want. I think we should default to stable though, and have a recurring job that tests edge.

Reply:
Yes, we should do a test from edge before releasing microk8s.